ralph/ralph.sh

```
@@ -1,119 +1,463 @@
+  }
+
+  # Count stories
+  TOTAL=$(jq_or_default '.userStories | length' 0 "$PRD_FILE")
+  TOTAL=${TOTAL:-0}
+  if [ "$TOTAL" -eq 0 ]; then
+    PASSED=0; FAILED=0; PCT=0; FILLED=0; EMPTY=40
   else
-    # Claude Code: use --dangerously-skip-permissions for autonomous operation, --print for output
-    OUTPUT=$(claude --dangerously-skip-permissions --print < "$SCRIPT_DIR/CLAUDE.md" 2>&1 | tee /dev/stderr) || true
+    PASSED=$(jq_or_default '[.userStories[] | select(.passes == true)] | length' 0 "$PRD_FILE")
+    FAILED=$((TOTAL - PASSED))
```

Shellcheck (0.11.0)

[warning] FAILED appears unused. Verify use (or export if used externally).

(SC2034)